src/utils/fixtures.ts

```diff
@@ -9,3 +10,13 @@ export const MOCK_MEMBER = {
   updatedAt: new Date(),
   type: MemberType.Individual,
 };
+
+export const MOCK_MEMBER_RECORD = {
+  id: 'id',
+  name: 'creator',
+  email: 'email',
+  extra: {},
+  createdAt: new Date(),
+  updatedAt: new Date(),
+  type: MemberType.Individual,
+} as MemberRecord;
```

Comment:
You should not cast an object to a MemberRecord. You should use the `convertJs` function from sdk to convert from a JS object to an immutable and set the type of the variable to `MemberRecord`.